.travis.yml

```diff
@@ -32,19 +36,14 @@ script:
         make &&
         sudo make install &&
         sudo ldconfig )
```

Comment:
`sudo ldconfig` is now useless since we don't use the installed binaries.

We also run `sudo ldconfig` in the command before this one (line 32) and it useless as well. But maybe we should consider running tests on both a) uninstalled binaries and b) installed binaries (in order to make sure we install them at the proper place).